lib/component.js

```diff
@@ -68,11 +69,29 @@ function build ( string, opts, readFile ) {
 						promises.push( readFile( src ).then( str => partials[id] = str ) );
 					}
 				} else if ( !type || type === 'text/javascript' || type === 'application/javascript' ) {
+					const rel = getAttr( 'rel', item);
```

Comment:
I was mostly just going for highlighting support with the script tag, I don't think there's an reason it couldn't be in a style tag beyond that. I have a few ideas for components that would generate chunks of styles based on config e.g. a grid framework with customizable units (I _need_ 14ths on this project because _reasons_). In that case the style fn would look much less like css as in the example and more like js.